custom-widget-builder/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "custom-widget-builder",
+  "version": "0.0.1",
+  "description": "Bonitasoft UI Designer: Custom Widget Builder",
+  "main": "build/src/starter.js",
+  "license": "GPLv2",
```

Comment:
Why not MIT ?

Reply:
We had this discussion already for BDE and web components... And the response at that time was to use GPLv2. We can re-discuss the licence with management, but for now I'm afraid we have to keep like this.